src/Utils/Strings.php

```diff
@@ -332,10 +332,10 @@ public static function compare($left, $right, $len = NULL)
 	 * @param  string|array
 	 * @return string
 	 */
-	public static function findPrefix($strings)
+	public static function findPrefix(...$strings)
 	{
-		if (!is_array($strings)) {
-			$strings = func_get_args();
+		if (is_array($strings[0])) {
+			$strings = $strings[0];
```

Comment:
This is not so simple because it changes signature. Previously the signature suggested that `findPrefix(['bar', 'baz'])` is preferred. Now it suggests that `findPrefix('bar', 'baz')` is preferred. Do we really want to prefer the variadic syntax over normal array? Passing directly array is slightly faster on PHP 7. AFAIK we should pick one syntax and deprecate the other.